conf/messages.en

```diff
@@ -362,17 +362,26 @@ suitability.noNeedToRegister.p4 = You can
 suitability.noNeedToRegister.p4.link = return to register and maintain a trust for a client
 
 nonTaxableTrustRegistrationAccessCode.title = Private Beta
-nonTaxableTrustRegistrationAccessCode.heading = Private Beta
-nonTaxableTrustRegistrationAccessCode.p1 = In order to control the number of Non-Taxable Trust registrations, we have added limited access for the private beta. You will need to enter your provided access code and select ''Continue'' to enter the registration journey.
+nonTaxableTrustRegistrationAccessCode.caption = Private Beta
+nonTaxableTrustRegistrationAccessCode.heading = Register a non-taxable trust
+nonTaxableTrustRegistrationAccessCode.p1 = You can now use the Trust Registration Service to register non-taxable trusts. To register a non-taxable trust, enter the access code and select 'continue'
```

Comment:
The `'` style of apostrophe needs to be escaped with another one. So it should be `''continue''`

Reply:
Updated